noxfile.py

```diff
@@ -198,9 +198,9 @@ def tests(session: nox.sessions.Session):
 
 
 @nox.session(python=PY_VER, venv_backend="conda")
-def gallery(session: nox.sessions.Session):
+def doctest(session: nox.sessions.Session):
     """
-    Perform iris gallery doc-tests.
+    Perform iris gallery and doc-tests.
```

Comment:
Would it not make more sense to run the gallery tests *after* the docs-build ?
(or does that leave a mess behind?)

Reply:
@pp-mo The doctests require to change directory but the gallery doesn't, so I erred on the sided of doing the gallery first, then change directory... I'll play and see, but yeah perhaps the doctests are more likely to fail, so it might be more economical to do them first :+1: